gameoflife.php

```diff
@@ -59,6 +70,17 @@
     $outputs->addOptions($options);
 }
 
+foreach (glob("rules/*.php") as $file)
+{
+    $baseName = basename($file, ".php");
+    $className = "GameOfLife\\rules\\" . $baseName;
+    if ($baseName == "BaseRule") continue;
+    if (!class_exists($className)) continue;
+    $rules = new $className;
```

Comment:
I was confused that this was named `$rules` because it only contains one rule. So `$rule` would be better as name.